tests/Microsoft.Graph.DotnetCore.Core.Test/Requests/Content/BatchResponseContentTests.cs

```diff
@@ -30,7 +30,7 @@ public async Task BatchResponseContent_InitializeWithNoContentAsync()
         [Fact]
         public async Task BatchResponseContent_InitializeWithEmptyResponseContentAsync()
         {
-            string jsonResponse = "{ responses: [] }";
+            string jsonResponse = "{ \"responses\": [] }";
```

Comment:
>\"responses\" [](start = 37, length = 13)

was this a prior issue with the test or something you had to do with the change.  What I'm getting it as that I would have expected test content not to need a change if all things were equal.

Reply:
System.text.json is more strict with the json standard. Therefore, properties without the double quotes are not valid json.